styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -339,20 +349,17 @@ public void create(Environment environment) {
 
     final RateLimiter dequeueRateLimiter = RateLimiter.create(DEFAULT_SUBMISSION_RATE_PER_SEC);
 
-    final OutputHandler[] outputHandlers = new OutputHandler[] {
+    outputHandlers.addAll(ImmutableList.of(
```

Comment:
Populating `outputHandlers` right after variable declaration would be a bit clearer.

Reply:
See above.